modules/libcom/RTEMS/posix/rtems_init.c

```diff
@@ -938,6 +929,20 @@ POSIX_Init ( void *argument __attribute__((unused)))
     struct timespec  	now;
     char timeBuff[100];
 
+    /*
+     * If RTEMS is used with the POSIX API, the init task
+     * 'POSIX_Init()' is unfortunately given the priority '2'. This
+     * corresponds to the second lowest POSIX prio (RTEMS pthread
+     * prio 256). This task shoud have IOCsh prio.
+     */
+
+    /* PosixMaxPrio == 100 */
+    rtems_task_priority mainPrio = RTEMS_MAXIMUM_PRIORITY - ( RTEMS_MAXIMUM_PRIORITY - RTEMS_MAXIMUM_PRIORITY) * epicsThreadPriorityIocsh / 100;
```

Comment:
Core Group:

`RTEMS_MAXIMUM_PRIORITY - RTEMS_MAXIMUM_PRIORITY` = 0.

This needs a little more testing.